omnistrate.pro.yaml

```
@@ -2252,7 +2326,8 @@
     depends_on:
       - cluster-sz
       - cluster-sz-rebalance
-    
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)